docs/data/migration/migration-tree-view-v6/migration-tree-view-v6.md

```diff
@@ -329,6 +329,34 @@ you can use the new `onNodeSelectionToggle` prop which is called whenever a node
 
 :::
 
+### Focus the Tree Item instead of the Tree View
```

Comment:
I think each breaking change should have a **why** section, as a mean to help sell the value to developers who will surfer doing the change.